src/main/java/pl/scoutCamp/api/controller/rest/RankingRestController.java

```diff
@@ -52,6 +74,15 @@ private List<TeamCategorizationSheetDTO> getAllTeamCategorizationSheetsDTO(Strin
         return getSortedTeamCategorizationSheetsDTO(teamsByPeriod, sortedBy);
     }
 
+    private List<TeamCategorizationSheetDTO> getAllTeamCategorizationSheetsDTO(Integer regimentId, String periodName, String sortedBy) {
+        List<TeamCategorizationSheetDTO> teamsByPeriodAndRegiment = new ArrayList<>(teamCategorizationSheetService
```

Comment:
For readibility purposes IMO it would be better to move these `teamCategorizationSheetService.findTeams...` operations to separate variable and then pass it to the ArrayList initializer